tests/test_filepath.py

```diff
@@ -147,20 +144,18 @@ def test_filepath_class_s3_again():
 def test_filepath_cleanup(table=Filepath(), store="repo"):
     """test deletion of filepath entries from external table """
     stage_path = dj.config['stores'][store]['stage']
-    filename = 'file.dat'
     n = 20
     contents = os.urandom(345)
     for i in range(n):
-        relative_path = os.path.join(*random.sample(('one', 'two', 'three', 'four'), k=3))
-        os.makedirs(os.path.join(stage_path, relative_path), exist_ok=True)
-        managed_file = os.path.join(stage_path, relative_path, filename)
-        with open(managed_file, 'wb') as f:
-            f.write(contents)   # same contents in all the files
+        relative_path = Path(*random.sample(('one', 'two', 'three', 'four'), k=3))
+        managed_file = Path(stage_path, relative_path, 'file.dat')
+        managed_file.parent.mkdir(parents=True, exist_ok=True)
+        with managed_file.open('wb') as f:
+            f.write(contents)  # same in all files
         table.insert1((i, managed_file))
```

Comment:
This test failed on WIN10. Error details: `datajoint.errors.DuplicateError: ("Duplicate entry '1' for key 'PRIMARY'", 'To ignore duplicate entries in insert, set skip_duplicates=True')`

Reply:
this happens sometimes when the previous test run was terminated before cleanup